### PR TITLE
added blog for principle four

### PR DESCRIPTION
**Pull request by beck410**
@neolytics @alexandralphillips

Is this post ok for the series?

Could you  take a look at the blog post and check for errors etc.

### CODE REVIEW

File: _posts/2015-01-28-make-data-easy-to-find-and-use.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Nashville is well on its way to adopting this principle. (The Nashville Open Data Portal)[https://data.nashville.gov/] makes data collected by the Nashville and Davidson Country Departments available to the public. Its website is easy to navigate and it has a range of options for exporting datasets. Datasets range from art grants and historic markers to General Government employees titles and base annual salaries. Code For Nashville is working with the Nashville mertro government to prioritize which datasets are released first and in what format. Through the magic of (Socrata)[http://www.socrata.com/] the public are able to view the datasets as visual maps, graphs and use filters to manuipulate the information.
```
Comment (jchapin) [inline _posts/2015-01-28-make-data-easy-to-find-and-use.markdown]: Typo: mertro > metro

Comment (jchapin) [inline _posts/2015-01-28-make-data-easy-to-find-and-use.markdown]: Typo: manuipulate > manipulate

**Comment by jchapin** [comment]
@beck410, the content looks great. I found a couple of typos, but that's all I would fix.

**Comment by beck410** [comment]
thanks @jchapin 

Types are fixed.

I'll wait until principle 3 is online before i merge

**Pull request merged**